all/entrypoint.sh

```diff
@@ -432,6 +443,10 @@ if [[ $GEN_RBI == true ]]; then
     GEN_STRING="$GEN_STRING --rbi_out=$OUT_DIR"
 fi
 
+if [[ $GEN_MYPY == true ]]; then
+    GEN_STRING="$GEN_STRING --mypy_out=$OUT_DIR"
```

Comment:
should `--mypy_grpc_out` be used here as well?
https://github.com/nipunn1313/mypy-protobuf#grpc